packages/multimedia/zvbi/package.mk

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2020-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="zvbi"
+PKG_VERSION="0.2.35"
+PKG_SHA256="fc883c34111a487c4a783f91b1b2bb5610d8d8e58dcba80c7ab31e67e4765318"
+PKG_LICENSE="GPL"
+PKG_SITE="http://zapping.sourceforge.net/ZVBI"
+PKG_URL="http://mirrors.kodi.tv/build-deps/sources/${PKG_NAME}-${PKG_VERSION}.tar.bz2"
```

Comment:
```suggestion
PKG_URL="https://downloads.sourceforge.net/sourceforge/zapping/zvbi-${PKG_VERSION}.tar.bz2"
```

Reply:
I hate sourceforge :smile: What's the benefit of sourceforge over the same source the add-on that uses zvbi is using?

Reply:
Everyone hates sourceforge, I won't contradict but using some "random" mirror instead of the official source is also not really a good idea from my pov. Sadly no proper github mirror is there, but thats a usual situation for those near death linux libs.